app/libs/controller_crud.php

```diff
@@ -4,16 +4,20 @@
  * Parent Controller to manage the database with the
  * new ActiveRecord
  */
-class ControllerCrud extends AdminController
+abstract class ControllerCrud extends AdminController
 {
 
-    /** @var string Nombre del modelo en CamelCase */
+    /** @var string Folder in views/_shared/scaffolds/ */
+    public $scaffold = 'kumbia';
+    /** @var string Model Name in CamelCase */
     public $model = '';
+    /** @var int Number of records per page */
+    const PER_PAGE = 30;

```

Comment:
We decide for the user in the UI ??
A constant is that a constant.

Reply:
Ok... In the config.php it would not allow a customization by controller, I will add it as a public attribute: $perPage